src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx

```diff
@@ -193,6 +195,33 @@ describe("when pressing bid button", () => {
   })
 })
 
+describe("editing bid amount", () => {
+  it("allows you to go to the max bid edit screen and select a new max bid", () => {
+    const fakeNavigator = new FakeNavigator()
+    const fakeNavigatorProps = {
+      ...initialPropsForRegisteredUser,
+      navigator: fakeNavigator,
+    }
+    fakeNavigator.push({ component: ConfirmBid, id: "", title: "", passProps: fakeNavigatorProps })
+
+    const component = renderer.create(<ConfirmBid {...initialPropsForRegisteredUser} navigator={fakeNavigator} />)
+
+    const selectMaxBidRow = component.root.findAllByType(TouchableWithoutFeedback)[0]
+    expect(selectMaxBidRow.findByType(Serif16).props.children).toEqual("$45,000")
+    selectMaxBidRow.instance.props.onPress()
+
+    const editScreen = fakeNavigator.nextStep()
+    expect(editScreen.root.type).toEqual(SelectMaxBidEdit)
+    expect(editScreen.root.props.selectedBidIndex).toEqual(0)
+    editScreen.root.instance.setState({ selectedBidIndex: 1 })
+
+    editScreen.root.findByType(Button).instance.props.onPress()
+
+    const confirmScreen = fakeNavigator.nextStep()
+    expect(confirmScreen.root.type).toEqual(ConfirmBid)
```

Comment:
I really wanted to test that the `confirmScreen` here displays `$46,000`, but I wasn't sure how to do that in the test environment. Basically, calling `fakeNavigator.nextStep()` to return the next component passes in the component's _original_ props, thus overriding what was previously changed.

Maybe you can look at this with me on Monday? If it's testable I would love to add that.

Reply:
You could possibly do the same as the line `210`, but make sure to `find` after `onPress()` since the `selectMaxBidRow` variable may be referencing a slate object. Happy to chat about this on Monday.